release_notes/364-release-notes.adoc

```diff
@@ -0,0 +1,57 @@
+[id="364-release-notes"]
+= Red Hat Advanced Cluster Security for Kubernetes 3.64
+include::modules/common-attributes.adoc[]
+:context: 364-release-notes
+
+toc::[]
+
+{product-title} 3.64 includes feature enhancements, bug fixes, scale improvements, and other changes.
+
+//To upgrade to this release from a previous version, see the link:/docs/upgrade-stackrox/[Upgrade]section.
+
+*Release date:* August 11, 2021
+
+[id="new-features"]
+== New features
+
+* *ROX-7230*: You can now use deployment and namespace annotations to define where {product-title} sends the violation notifications when configuring your notifiers such as Slack, Microsoft Teams, Email, and others.
```

Comment:
```suggestion
* *ROX-7230*: You can now use deployment and namespace annotations to define where {product-title} sends violation notifications when configuring your notifiers. Notifications can be sent to Slack, Microsoft Teams, Email, and others.
```
(I think that's what the list of communications options is included for, lmk if that's incorrect)